var/spack/repos/builtin/packages/axel/package.py

```diff
@@ -10,18 +10,13 @@ class Axel(AutotoolsPackage):
     """Axel is a light command line download accelerator for Linux and Unix"""
 
     homepage = "https://github.com/axel-download-accelerator/axel"
-    url      = "https://github.com/axel-download-accelerator/axel/archive/v2.16.1.tar.gz"
+    url      = 'https://github.com/axel-download-accelerator/axel/releases/download/v2.17.10/axel-2.17.10.tar.bz2'
 
-    version('2.16.1', sha256='64529add74df3db828f704b42d4ec3fcdacb8142c84f051f9213637c337e706c')
+    version('2.17.10', sha256='c0d26eba6b94945cd98c5b69ca6df2744639d17bfd49047ef51a8a48f067de10')
+    version('2.16.1',  sha256='763066efc61e4f7be2eb59afa049bdbc520837e01c95a78f403e542ad82f2719')
 
-    depends_on('m4', type='build')
-    depends_on('autoconf', type='build')
-    depends_on('automake', type='build')
-    depends_on('libtool', type='build')
-    depends_on('pkgconfig', type='build')
-    depends_on('gettext')
+    depends_on('gettext', type='build')
```

Comment:
`pkgconfig` is still required:
```
configure: error: The pkg-config script could not be found or is too old.  Make sure it is in your PATH or set the PKG_CONFIG environment variable to the full path to pkg-config.
```
Moreover, `gettext` seems to be a proper dependency, not only a build dependency.

Reply:
No, it appears not, I even force-uninstalled at gettext builds I had to verify:
```bash
spack load axel@2.16.1
ldd `which axel`|grep -e gettext -e intl
```
<no output> and looking up the dependencies I came up empty.

Pushed keeping pkg-config (I thought I didn't install it into my containers but boost development packages pulled pkg-config, not uninstalled again)

Reply:
> No, it appears not, I even force-uninstalled all my gettext builds I had to verify and I ran the executable then, and ldd:

That's most likely because your libc includes the necessary functions:
```
checking for GNU gettext in libc... yes
```
On systems where this is not the case, the build will probably fail. It's also listed here as a dependency: https://github.com/axel-download-accelerator/axel#dependencies